package.json

```diff
@@ -18,6 +18,7 @@
     "postinstall": "next build --no-lint && nsm build --skip-build --only-api-files && npm run type-css",
     "format": "prettier --write --ignore-unknown .",
     "format:check": "prettier --check --ignore-unknown .",
+    "format:version": "prettier --version",
```

Comment:
Suggestion:  Let's file an issue to remove this and the `prettier --version` in pr.yml in a few weeks if the problem doesn't recur.

Reply:
...looks like this was discussed on slack already. 😃